examples/rust-cross-program-invocation/src/entrypoint.rs

```diff
@@ -0,0 +1,16 @@
+//! Program entrypoint
+
+#![cfg(all(target_arch = "bpf", not(feature = "no-entrypoint")))]
```

Comment:
I think we can get away with this:
```suggestion
#![cfg(feature = "no-entrypoint")]
```

Reply:
Not happy with just `not(feature = "no-entrypoint")`, `global_allocator` conflicts with the one in solana's `measure` crate.  Seems like the checks in the entrypoint which already test for bpf are not working...

Reply:
ah, right.  Under `cargo test` measure gets sucked in.  Something to fix later... :(